### DISCUSSION TITLE
Access protection for files.publii.jason

### DISCUSSION THREAD

**Discussion by carstenfeller** [GetPublii/Publii#1821] [Help] [answered]
Hi there,
I'm wondering what _"Consider access protection for this file in your server configuration **if required**."_ exactly means?

![files_json](https://github.com/user-attachments/assets/b38f45f8-f24d-4a5b-87ea-68df21339576)

The file is accesible via browser. 

![files_json-3](https://github.com/user-attachments/assets/8f1e680e-5a93-427f-9147-6546e7a82500)

Should this be restricted?

Or are we talking about the file permissions?

![files_json-2](https://github.com/user-attachments/assets/c4f2623d-7570-481a-8548-c27df82332cb)

Thanks in advance.
Best regards.
Carsten

**Comment by bobmitro** [comment, marked as answer]
Please check out this article: [Recommended Server Settings](https://getpublii.com/docs/recommended-server-settings.html#protectingthefilespubliijsonfile)
